### DISCUSSION TITLE
FYI possible issue with newest Artery ISP Programmer V2.0.07

### DISCUSSION THREAD

**Discussion by roguestreak** [keirf/flashfloppy#736] [General]
FYI I've just gotten hold of a Gotek with AT415 and been trying to program the bootloader, however (for me?) the current V2.0.07 cannot get past the second screen as there are no buttons down the bottom (Next etc.)  There's a slight scroll, but nothing appears.
I followed a link elsewhere and found V1.5.46 (downloads from the ArteryTek website, so it's an unlisted old version), and everything worked as it should.
So if anyone comes across the same issue, try tracking down an older version; here's the one I downloaded and successfully used
https://www.arterytek.com/download/Artery_ISP_Programmer_V1.5.46.zip

**Comment by keirf** [comment]
Thanks for the hint. v2.0.07 is working for me, but perhaps others will have the same problem as you.

**Reply by roguestreak** [reply]
I later saw someone post in the thread I found that link, that they had an issue with the programmer not showing the buttons.  They didn't mention a version, however said they were using Win11, and when tried on Win10 it worked; I'm on Win10, so either it's a different version or there's a bunch of variables that can effect things.
So just a heads up that the programmer can potentially be flakey, and to look around for older versions if you need to get around the issue.

**Reply by keirf** [reply]
That is weird. They are always mucking with the programmer unfortunately, and also it's not signed. So the user experience is not wonderful.

**Comment by mattg1974** [comment]
I have the same issue here with windows 10. I solved it by using the multi prorgrammer just one or two links below and it worked for me.
